### PR TITLE
add 6.4-Change-Folder-Sync.md

### PR DESCRIPTION
**Pull request by charlesweird**

### CODE REVIEW

File: user-guide/6.4-Change-Folder-Sync.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Returns summary details of the folder as an confirmation for folder sync flag update completion.
+
+### Error responses:
+| Error Code | Description |
+|-----|-----|
+| *invalid_resource* | Specified folder does not exist. |
+| *invalid_field_value* | The "sync" property has invalid value. |
+
+### Example requests:
+```
```
Comment (kirvin) [inline user-guide/6.4-Change-Folder-Sync.md]: I think we can go ahead and make this github markdown (GHFM):
` ` ` bash

**Comment by kirvin** [comment]
@charlesweird other than marking the code blocks with the appropriate language, this looks good.

File: user-guide/6.4-Change-Folder-Sync.md (hunk 1 above)
Comment (charlesweird) [inline user-guide/6.4-Change-Folder-Sync.md]: @kirvin thanks, I will do that

**Comment by kirvin** [comment]
We'll merge this pull request after the business-36 release goes live

**Comment by charlesweird** [comment]
:thumbsup:

**Pull request merged**